coral-dbt/src/main/resources/tests/test_incremental.py

```diff
@@ -0,0 +1,78 @@
+#
+# Copyright 2023 LinkedIn Corporation. All rights reserved.
+# Licensed under the BSD-2 Clause license.
+# See LICENSE in the project root for license information.
+#
+import subprocess
+import unittest
+from jinja2 import Template
+
+SEEDS_DIR = 'seeds/'
+URL = 'http://localhost:8080/api/incremental/rewrite'
+
+class TestUtils:
+    @staticmethod
+    def construct_generate_incremental_code_test(name):
+        output_table = name + '_output'
+        expected_file = open(SEEDS_DIR + 'test_' + name + '_expected.txt', 'r')
+        expected_incremental_spark_scala = expected_file.read()
+        expected_file.close()
+
+        return({
+            'output_table': output_table,
+            'expected_incremental_spark_scala': expected_incremental_spark_scala,
+        })
+
+    @staticmethod
+    def test_generate_incremental_code(coral_response, table_names, output_table):
+        with open('generate_incremental_code.j2') as f:
+            template = Template(f.read())
+
+        context = {'coral_response': coral_response, 'table_names': table_names, 'output_table': output_table}
+        output = template.render(context)
+
+        return output
+
```

Comment:
as we add more tests, this file might become unmanageable. 

can you evaluate if splitting it up is an option? maybe Utils + setup & teardown can be in a separate file, tests separate.

Reply:
Hmm, I feel like it makes sense to keep all of these in one file since this is all setup/teardown and utils associated with testing incremental, which can be likened to the beforeClass() and afterClass() methods and helper functions in the tests in the Java modules in Coral.